engine/api/api_routes.go

```diff
@@ -155,19 +155,10 @@ func (api *API) InitRouter() {
 	r.Handle("/project/{key}/application/{permApplicationName}/branches", r.GET(api.getApplicationBranchHandler))
 	r.Handle("/project/{key}/application/{permApplicationName}/vcsinfos", r.GET(api.getApplicationVCSInfosHandler))
 	r.Handle("/project/{key}/application/{permApplicationName}/remotes", r.GET(api.getApplicationRemoteHandler))
```

Comment:
"r.Handle("/project/{key}/application/{permApplicationName}/remotes", r.GET(api.getApplicationRemoteHandler))" to delete, no?

Reply:
and class ApplicationWorkflowService ?